python/tvm/relay/frontend/keras.py

```diff
@@ -147,7 +147,21 @@ def _convert_advanced_activation(inexpr, keras_layer, etab):
 def _convert_merge(inexpr, keras_layer, _):
     merge_type = type(keras_layer).__name__
     ret = inexpr[0]
-    if merge_type == 'Subtract':
+    if merge_type == 'Dot':
+        if keras_layer.axes == 1:
+            ret = _op.transpose(ret, axes=[0, 2, 1])
+            inexpr[1] = _op.transpose(inexpr[1], axes=[0, 2, 1])
+
+        elif isinstance(keras_layer.axes, list):
+            if keras_layer.axes == [1, 2]:
+                ret = _op.transpose(ret, axes=[0, 2, 1])
+            elif keras_layer.axes == [2, 1]:
+                inexpr[1] = _op.transpose(inexpr[1], axes=[0, 2, 1])
+            else:
+                raise tvm.error.OpAttributeUnimplemented(
+                    'Dot with {} is not supported.'.format(keras_layer.axes))
```

Comment:
How about the following?
 - `axes=[axes,axes]` when `axes` is an integer.
 - raise an error when axes[i] is neither 1 nor 2.
 - transpose inexpr[i] when axes[i] is 1.

This can also handle `axes=[1,1]` and raise an error against `axes=0` or `axes=3`.

Reply:
@kazum  thanks, I like the idea!